src/basic.mli

```diff
@@ -4,9 +4,15 @@
 (* Copyright (C) --- OCamlPro --- See README.md for information and licensing *)
 (******************************************************************************)
 
+
+(** The main entry point of the library.
+    It provides a functor building each key module of OcplibSimplex. *)
+
 module Make
     (Var : ExtSigs.VAR_SIG)(R : ExtSigs.R_SIG)(Ex : ExtSigs.EX_SIG) : sig
```

Comment:
A few problems:
- all of the `_SIGS` seems highly redundant (particularly since they come from the `ExtSigs` module)
- once stripped of the unneeded `_SIGS`, some of the signature names are really obscure, and it'd be better to rename them to something more readable, e.g. `R_SIG` -> `Rationals`, `EX_SIG` -> `Explanation` in this case.